README.md

```diff
@@ -6,37 +6,230 @@ rocDecode is a high-performance video decode SDK for AMD GPUs. Using the rocDeco
 access the video decoding features available on your GPU.
 
 > [!NOTE]
-> The published documentation, including installation steps, is available at [rocDecode](https://rocm.docs.amd.com/projects/rocDecode/en/latest/index.html) in an organized, easy-to-read format, with search and a table of contents. The documentation source files reside in the `rocDecode/docs` folder of this repository. As with all ROCm projects, the documentation is open source. For more information on contributing to the documentation, see [Contribute to ROCm documentation](https://rocm.docs.amd.com/en/latest/contribute/contributing.html).
+> The published documentation is available at [rocDecode](https://rocm.docs.amd.com/projects/rocDecode/en/latest/index.html) in an organized, easy-to-read format, with search and a table of contents. The documentation source files reside in the `rocDecode/docs` folder of this repository. As with all ROCm projects, the documentation is open source. For more information on contributing to the documentation, see [Contribute to ROCm documentation](https://rocm.docs.amd.com/en/latest/contribute/contributing.html).
 
-After installation, the rocDecode files are found in the following locations:
+## Supported codecs
 
-* Libraries: `/opt/rocm/lib`
-* Header files: `/opt/rocm/include/rocdecode`
-* Samples folder: `/opt/rocm/share/rocdecode`
-* Documents folder: `/opt/rocm/share/doc/rocdecode`
+* H.265 (HEVC) - 8 bit, and 10 bit
+* H.264 (AVC) - 8 bit
+* AV1 - 8 bit, and 10 bit
 
-You can verify your installation using either the `rocdecode-test` package or by running a sample.
+## Prerequisites
 
+* Linux distribution
+  * Ubuntu - `20.04` / `22.04`
+  * RHEL - `8` / `9`
+  * SLES - `15 SP5`
+
+* [ROCm-supported hardware](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html)
+> [!IMPORTANT] 
+> `gfx908` or higher GPU required
+
+* Install ROCm `6.1.0` or later with [amdgpu-install](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/how-to/amdgpu-install.html): Required usecase - rocm
+> [!IMPORTANT]
+> `sudo amdgpu-install --usecase=rocm`
+
+* [Video Acceleration API](https://en.wikipedia.org/wiki/Video_Acceleration_API) Version `1.5.0` or later - `Libva` is an implementation for VA-API
   ```shell
-  mkdir rocdecode-test && cd rocdecode-test
-  cmake /opt/rocm/share/rocdecode/test/
-  ctest -VV
+  sudo apt install libva-dev
+  ```
+> [!NOTE]
+> RPM Packages for `RHEL`/`SLES` - `libva-devel`
+
+* AMD VA Drivers
+  ```shell
+  sudo apt install mesa-amdgpu-va-drivers
+  ```
+
+* CMake Version `3.5` or later
```

Comment:
@AryanSalmanpour - we may have to update this PR once this is merged to use AMD Clang